lega/ingest.py

```diff
@@ -72,9 +83,8 @@ def work(fs, inbox_fs, channel, data):
     LOG.debug('Opening %s', filepath)
     with inbox.open(filepath, 'rb') as infile:
         LOG.debug(f'Reading header | file_id: {file_id}')
-        beginning, header = get_header(infile)
-
-        header_hex = (beginning+header).hex()
+        header_bytes = get_header(infile)
+        header_hex = header_bytes.hex()
         data['header'] = header_hex
```

Comment:
I realise that this is not a change now. But I want to highlight it. This means that we send the header over the message broker. Do we _really_ want to do that?

Reply:
There is then no need for a database lookup

Reply:
Yes. But on the other hand, the header might end up in a logfile somewhere or in some message it shouldn't. I don't think this should be changed now. It's just something that needs to be evaluated.

Reply:
There is a need for a database lookup when the cancelation feature is implemented. Before running the verify job, we should check if we should. Therefore, the header will be removed from the message, and the verify job will fetch it from the database (and check for cancelation).
Like you wrote, this is not for that PR.